Stratis.Bitcoin/BlockPulling/BlockPuller.cs

```diff
@@ -235,14 +240,13 @@ private void AssignPendingVectors()
         }
 
         /// <inheritdoc />
-        public bool IsDownloading(uint256 hash)
+        public void CheckBlockStatus(uint256 hash, out bool IsDownloading, out bool IsReady)
```

Comment:
Is there any actual benefit to it? As you know my style of writing things is to avoid using these new things as they are not readable for many coders. Yes, you can read them, that is fine, but there can be many people that are not happy to read them. This is why I would never use it myself. It is just a syntactical sugar that is not very hard to understand, but does not bring much and is just too new to be used.